src/components/Create/components/pages/NftRewards/hooks/useCreateFlowNftRewardsForm.ts

```diff
@@ -11,20 +12,7 @@ import {
 } from 'utils/nftRewards'
 import { useFormDispatchWatch } from '../../hooks'
 
-type NftRewardsFormProps = Partial<{
-  rewards: NftRewardTier[]
-  collectionName?: string
-  collectionSymbol?: string
-  collectionDescription?: string
-  postPayMessage?: string
-  postPayButtonText?: string
-  postPayButtonLink?: string
-  onChainGovernance: JB721GovernanceType
-  useDataSourceForRedeem: boolean
-  preventOverspending: boolean
-}>
-
-export const useNftRewardsForm = () => {
+export const useCreateFlowNftRewardsForm = () => {
```

Comment:
**suggestion (non-blocking)** maybe this could be called "useNewProjectNftRewardsForm`